crates/release_plz_core/src/release.rs

```diff
@@ -131,9 +137,26 @@ fn release_package(
         info!("published {}", package.name);
     }
 
+    if input.gh_release {
+        publish_release(git_tag(&package.name, &package.version.to_string()));
+    }
+
     Ok(())
 }
 
+async fn publish_release(git_tag: String) {
+    let owner = RepoUrl::new("https://www.github.com").unwrap().owner;
+    let repo = RepoUrl::new("https://www.github.com").unwrap().name;
```

Comment:
The `unwrap` here will panic, because the string `"https://www.github.com"` doesn't have owner and name.
We need to find owner and name similarly to how we do it for the update command [here](https://github.com/MarcoIeni/release-plz/blob/36616bbb470e429c4686c41234f9a6b4b7dee936/crates/release_plz/src/args/update.rs#L86).
So we need to add `repo_url` to the input struct of the release [command](https://github.com/MarcoIeni/release-plz/blob/36616bbb470e429c4686c41234f9a6b4b7dee936/crates/release_plz/src/args/release.rs#L9)